lib/utils/status.js

```diff
@@ -1,10 +1,20 @@
 'use strict';
 
+const logger = require('webpack-log');
 const colors = require('./colors');
 const runOpen = require('./runOpen');
 
 // TODO: don't emit logs when webpack-dev-server is used via Node.js API
 function status(uri, options, log, useColor) {
+  if (options.quiet === true) {
```

Comment:
does this mean that there is now no way to silence these extra logs when using in node api?

Reply:
yes, this log should be outputted always, in future we will use webpack logger and you can filter this output

Reply:
that would be great. this seems like a regression IMO. we now have an extra set of console output. if possible when the webpack logger option is available would be great to link that to this issue.

Reply:
link to documentation? not sure what you mean.

Reply:
No docs right now, because it is hidden feature, look here https://github.com/webpack/webpack-dev-server/blob/master/lib/Server.js#L52

Reply:
@evilebottnawi

I am facing this as well. I use webpackbar and this extra console log noise actually make it weird because webpackbar tries to clear screen when showing the fancy bar

After this PR
<img width="611" alt="sad" src="https://user-images.githubusercontent.com/17883920/66816542-77f7b000-ef64-11e9-91dc-68319ac97ccd.PNG">

Before
<img width="497" alt="good" src="https://user-images.githubusercontent.com/17883920/66816667-a9707b80-ef64-11e9-928b-48fc8fc583c8.PNG">

Can we add an option to hide this output ? I'm sure many packages that use webpackbar is affected by this unknowingly :(

cc my team @yangshun

Reply:
pretty pretty please?  this output decreases dev experience IMO.

Reply:
Feel free to send a PR with new option

Reply:
This reverts #1486 and I believe it to _now_ be a breaking change.

> this log should be outputted always

Sorry, strongly disagree. What's the point of the option if it doesn't do what it says? I understand the custom logger is just a documentation issue right now, but it still points to `quiet` (and `noInfo`) being pointless/improperly named

Reply:
You actually can't fix this with a custom logger, because this setup noise is printed by `utils/status.js` directly, which is where this `log` override is happening. There's no intermediary step